sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2461,26 +2461,34 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
         s"but it's ${endSeconds.toString} now.")
   }
 
-  def failedToReadDeltaFileError(fileToRead: Path, clazz: String, keySize: Int): Throwable = {
+  def failedToReadDeltaFileError(fileToRead: Path, clazz: String, message: String): Throwable = {
     new SparkException(
-      errorClass = "_LEGACY_ERROR_TEMP_2258",
+      errorClass = "CANNOT_LOAD_STATE_STORE.CANNOT_READ_DELTA_FILE",
       messageParameters = Map(
         "fileToRead" -> fileToRead.toString(),
         "clazz" -> clazz,
-        "keySize" -> keySize.toString()),
+        "message" -> message),
```

Comment:
Please, put error message text to error-classes.json. As I can see `failedToReadDeltaFileError` is invoke twice with `message`:
1.
```scala
        throw QueryExecutionErrors.failedToReadDeltaFileError(
          fileToRead, toString(), s"$fileToRead does not exist"
        )
```

2.
```scala
          throw QueryExecutionErrors.failedToReadDeltaFileError(
            fileToRead, toString(), s"key size cannot be ${keySize}"
          )
```